examples/build.gradle

```diff
@@ -0,0 +1,30 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.3'
+    }
+}
+
+ext {
+    defaultCompileSdkVersion = 25
+    defaultMinSdkVersion = 9
+    defaultTargetSdkVersion = 25
+    defaultBuildToolsVersion = "25.0.2"
+    defaultSupportLibVersion = "25.1.0"
+    dagger_version = "2.6.1"
+}
+
+allprojects {
+    repositories {
+        jcenter()
+    }
+
+    // disable lint
+    tasks.whenTaskAdded { task ->
```

Comment:
hmm - why? If we have lint errors in the examples, we should fix them. We don't want examples that have bad programming practices